src/library/coverart.cpp

```diff
@@ -98,8 +97,8 @@ QImage CoverInfo::loadImage(
                 TrackFile(trackLocation),
                 pTrackLocationToken);
     } else if (type == CoverInfo::FILE) {
-        VERIFY_OR_DEBUG_ASSERT(!trackLocation.isEmpty()) {
-            kLogger.warning()
+        if (trackLocation.isEmpty()) {
```

Comment:
please add a comment that explains why this can be empty.

Reply:
Done